python/orca/example/learn/pytorch/brainMRI/README.md

```diff
@@ -0,0 +1,55 @@
+## Trouble Shooting
+- [x] no support of val set?
+- [ ] Difficult to identify a customized loss function; for example, dice coefficient
+- [ ] Unable to see the plot of matplotlib
```

Comment:
As plot results will be shown on the driver, it may be difficult to plot in a distributed fashion.